governance/handler.go

```diff
@@ -127,37 +156,62 @@ func (g *Governance) AddVote(key string, val interface{}) bool {
 
 func (g *Governance) adjustValueType(key string, val interface{}) interface{} {
 	k := GovernanceKeyMap[key]
-	reqType := GovernanceItems[k].t
 	var x interface{}
 
 	// When an int value comes from JS console, it comes as a float64
-	if reqType == uint64T && reflect.TypeOf(val) == float64T {
-		x = uint64(val.(float64))
-		if float64(x.(uint64)) == val.(float64) {
+	if GovernanceItems[k].checkValueType(uint64(0)) {
+		v, ok := val.(float64)
+		if !ok {
+			return val
+		}
+		x = uint64(v)
+		if float64(x.(uint64)) == v {
 			return x
 		}
 		return val
 	}
 
-	// address comes as a form of string from JS console
-	if reqType == addressT && reflect.TypeOf(val) == stringT {
-		if common.IsHexAddress(val.(string)) {
-			x = common.HexToAddress(val.(string))
+	// Otherwise, it comes as a string
+	v, ok := val.(string)
+	if !ok {
+		return val
+	}
+	if GovernanceItems[k].checkValueType(common.Address{}) {
+		addresses := strings.Split(v, ",")
+		switch len(addresses) {
+		case 0:
+			return val
+		case 1:
+			str := strings.Trim(v, " ")
+			if common.IsHexAddress(str) {
+				x = common.HexToAddress(str)
+				return x
+			} else {
+				return val
+			}
```

Comment:
Seems like it is a common pattern shown various area. What about making a function for this?

```go
func ConvertAddress(str string) (common.Address, error) {
  if common.IsHexAddress(str) {
    x = common.HexToAddress(str)
    return x
  } else {
    return val
  }
}

```

Reply:
Hmm.. I think the benefits of functionalization of this logic are unlikely to be significant. Looking at `case 1` and `case 2`, it is likely that many overlapping parts will occur again due to different logic.

Reply:
Ok then. Leave it as it is.